SolDefines.py

```diff
@@ -288,4 +288,22 @@ def solStructure(type_id):
         'structure':    '>III',
         'fields':       ['sol_version', 'solmanager_version', 'sdk_version'],
     },
+    {
+        'type':         SENS_MB7363_D2S2N1L1G1,

```

Comment:
please update:
- change names from `SENS_MB7363_D2S2N1L1G1` to `SOL_TYPE_SENS_MB7363_D2S2N1L1G1`
- add these values to the list of `SOL_TYPE_` values